tests/pytrec_eval_tests.py

```diff
@@ -5,6 +5,8 @@
 
 import pytrec_eval
 
+print(pytrec_eval)
+
```

Comment:
Looking through the pull requests and noticed this debug message was left in the unit tests